apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx

```diff
@@ -10,9 +10,7 @@ import { useTemplateController } from '../../../components/templates/use-templat
 import { StepActiveSwitch } from './StepActiveSwitch';
 
 const capitalize = (text: string) => {
```

Comment:
It's fine like this, not a big deal at all.  But I'm wondering why aren't we leaving it uncapitalised like we always did. I think it looks better and it fits the other identifiers (in the case of the channels) 
(email, SMS, in_app vs Email, Sms, In_app)

Reply:
because the enum is capitalized and if we do not do it here it will not work...

Reply:
Yeah sure we do not follow our standard for enum if we do that thu...